src/css.js

```diff
@@ -336,16 +333,23 @@ jQuery.each( [ "height", "width" ], function( i, name ) {
 		},
 
 		set: function( elem, value, extra ) {
-			var styles = extra && getStyles( elem );
-			return setPositiveNumber( elem, value, extra ?
-				augmentWidthOrHeight(
+			var matches = rcssNum.exec( value ),
```

Comment:
`matches` would be more clear (and [consistent](https://github.com/timmywil/jquery/blob/width-1712/src/css/adjustCSS.js#L6)) as `valueParts`.